Add "--process" option to override searched for process

laxsjo opened the pull request:
Adds an option that lets you separate the searched for process name from the command. This is to support commands which start another process, meaning that you can't find the original command using `pgrep`, like when using flatpaks.

This is based on the fork from https://github.com/academo/ww-run-raise/pull/8, so you should probably merge that before merging this.

BTW, feel free to completely change anything in this PR if any implementation details don't suit you, I just quickly threw this together in an afternoon. :)

academo commented:
thanks so much for the changes

I incorporated these changes here https://github.com/academo/ww-run-raise/commit/f0cb69aa49bece7b93181f368c43e74e1f1ae5a8

I didn't merge your PR because it had the kd6 script and many missing changes I did recently. I added you as a co-author in the commit I hope github picks you as a contributor.

laxsjo commented:
> thanks so much for the changes
> 
> I incorporated these changes here [f0cb69a](https://github.com/academo/ww-run-raise/commit/f0cb69aa49bece7b93181f368c43e74e1f1ae5a8)
> 
> I didn't merge your PR because it had the kd6 script and many missing changes I did recently. I added you as a co-author in the commit I hope github picks you as a contributor.

Oh it didn't seem to, but that's fine. Thank you anyway for trying to make sure that I did! :)
